datastore/tests/unit/test_client.py

```diff
@@ -934,6 +934,15 @@ def test_transaction_defaults(self):
             self.assertIs(xact, mock_klass.return_value)
             mock_klass.assert_called_once_with(client)
 
+    def test_read_only_transaction_defaults(self):
+        from google.cloud.datastore.transaction import Transaction
+        from google.cloud.datastore_v1.types import TransactionOptions        
+        creds = _make_credentials()
+        client = self._make_one(credentials=creds)
+        xact = client.transaction(read_only=True)
+        self.assertIsInstance(xact._options.read_only,
+                              TransactionOptions.ReadOnly)
```

Comment:
Just assert equality here. `__eq__` on protobuf objects will be defined as expected.